### DISCUSSION TITLE
Include sigv4 auth in alertmanager datasource

### DISCUSSION THREAD

**Discussion by hussain-ujjaini** [grafana/grafana#44360] [Data Sources]
Include sigv4 auth in alertmanager datasource to support aws prometheus alertmanager

<img width="566" alt="Screenshot 2022-01-24 at 1 31 35 PM" src="https://user-images.githubusercontent.com/34863783/150743948-83886aa6-135c-4088-8f2b-11513175deaf.png">

**Comment by taleena** [comment]
Issue: https://github.com/grafana/grafana/issues/44361

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
